ipc/Cargo.toml

```diff
@@ -13,7 +13,7 @@ log = "0.4"
 tokio-service = "0.1"
 jsonrpc-core = { version = "8.0", path = "../core" }
 jsonrpc-server-utils = { version = "8.0", path = "../server-utils" }
-parity-tokio-ipc = { git = "https://github.com/nikvolf/parity-tokio-ipc", branch = "stable" }
+parity-tokio-ipc = { git = "https://github.com/nikvolf/parity-tokio-ipc", branch = "master" }
```

Comment:
We should probably use a different branch since `master` might change to often CC @NikVolf